docs/guides/debug-qiskit-runtime-jobs.ipynb

```diff
@@ -0,0 +1,618 @@
+{
```

Comment:
<blockquote>However, you can use the&nbsp;<code>Neat</code>&nbsp;<code><a href="https://github.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.debug_tools.Neat#to_clifford" target="_blank">to_clifford</a></code>&nbsp;method to map tham to Clifford circuits for more efficient simulation.</blockquote><p> </p><p>There is a typo <code>tham</code>.</p><p> </p><blockquote>The&nbsp;<code><a href="https://github.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.debug_tools.Neat#to_clifford" target="_blank">to_clifford</a></code>&nbsp;method is a wrapper around the&nbsp;<code><a href="https://github.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.transpiler.passes.ConvertISAToClifford" target="_blank">ConvertISAToClifford</a></code>&nbsp;transpiler pass.</blockquote><p> </p><p>I'd add "... transpiler pass, <em>which can also be used independently</em>.", to give a reason why we mention it as a pass.</p><p> </p><blockquote>The simulation can be done in both ideal (<code><a href="https://github.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.debug_tools.Neat#ideal_sim" target="_blank">ideal_sim</a></code>) and noisy (<code><a href="https://github.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.debug_tools.Neat#noisy_sim" target="_blank">noisy_sim</a></code>) conditions. The simulated results support mathematical operations, and can therefore be compared with each other (or with experimental results) to calculate figures of merit.</blockquote><p> </p><p>The code below doesn't show these methods. This paragraph should be moved to Application 1 where they are actually used with context. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>